application/models/people_model.php

```diff
@@ -153,6 +153,14 @@ public function get_all_people() {
         }
     }
 
+    /*
+     * Get remained quota of RDB
+     */
+    public function get_rdb_quota() {
+        $query = $this->db->where('deleted', false)->where('rdb', true)->get('people');
+        return 80 - $query->num_rows();
```

Comment:
Instead of hard-coding, it is better to put the constant `80` in the config file.